src_new/util.js

```diff
@@ -1487,4 +1496,49 @@ exports.getDevicePlatform = function(){
 		refThis.logError("Unable to get device platform" , ex);
 	}
 	return deviceType;
-}
+};
+
+exports.updateAdUnitsWithEids = function(adUnits){
```

Comment:
updateAdUnitsWithEids: in this function, we are setting userIds and UserIdAsEids so naming it as "updateAdUnitsWithEids" looks like we are only adding Eids. 
Let's keep this function name as updateAdUnits so that in future if we have to make any other modification to adUnit we can add it in this function by calling the respective function from "updateAdUnits"

Reply:
Sure.
Done